### PR TITLE
Feature: Composable SVG

### PR DESCRIPTION
**Pull request by Suficio**

### CODE REVIEW

File: sources/standards/composability/container.move [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+        /// Type name of the witness authorized to withdraw the NFT
+        witness: TypeName,
+        /// ID of the NFT stored as a dynamic object field
+        nft_id: ID,
+    }
+
+    /// `ContainerDomain` object
+    struct ContainerDomain has store {
+        /// `ContainerDomain` ID
+        id: UID,
```
Comment (bausano) [inline sources/standards/composability/container.move:50]: Is this useful if the domain doesn't have `key` ability?

**Comment by Suficio** [comment]
It's used to store NFTs as dynamic fields, guess I should rename it to
`nfts` but that's already a field. 😅

File: sources/standards/composability/container.move (hunk 1 above)
Comment (Suficio) [reply sources/standards/composability/container.move:50]: It's used to store NFTs as dynamic fields, guess I should rename it to
`nfts` but that's already a field. 😅

Comment (bausano) [reply sources/standards/composability/container.move:50]: I see, yeah I'd rename it or add a comment

**Review by bausano — APPROVED** [review]

File: sources/standards/composability/container.move (hunk 1 above)
Comment (nmboavida) [reply sources/standards/composability/container.move]: Renamed it to `NftBagDomain`

**Pull request merged**